apps/rest_api/lib/routers/public.ex

```diff
@@ -28,4 +28,10 @@ defmodule RestAPI.Routers.Public do
       end
     end
   end
+
+  scope "/admin/v1", RestAPI.Controller.Admin do
+    # pipe_through :authenticated
+
+    resources("/users", User, except: [:new])
+  end
```

Comment:
We will move it to another router later but it's ok to be here for a now.
Just remember to uncomment this line (This request has to be authenticated).

Reply:
In a near future we will also authorized admin requests.